{"name": "test_delete_courier_with_nonexistent_id", "status": "failed", "statusDetails": {"message": "AssertionError: Сообщение об ошибке не совпадает\nassert 'Курьера с таким id нет.' == 'Курьера с таким id нет'\n  \n  - Курьера с таким id нет\n  + Курьера с таким id нет.\n  ?                       +", "trace": "self = <test_scooter_api.TestCourierDeletionAPI object at 0x105d78f50>\n\n    @allure.feature(\"Удалить курьера\")\n    @allure.story(\"2. Если отправить запрос с несуществующим id, вернётся ошибка.\")\n    @allure.description(\n        \"Проверка, что система вернёт ошибку, если указать несуществующий ID курьера\"\n    )\n    @allure.step(\"Отправляем DELETE-запрос на удаление курьера с несуществующим ID\")\n    def test_delete_courier_with_nonexistent_id(self):\n        response = requests.delete(f\"{Url.BASE_URL}/courier/{self.courier_id}\")\n        print(\n            f\"Статус после попытки удаления несуществующего курьера: {response.status_code}\"\n        )\n        assert (\n            response.status_code == 404\n        ), \"Код ответа не 404 при удалении несуществующего ID\"\n>       assert (\n            response.json().get(\"message\") == \"Курьера с таким id нет\"\n        ), \"Сообщение об ошибке не совпадает\"\nE       AssertionError: Сообщение об ошибке не совпадает\nE       assert 'Курьера с таким id нет.' == 'Курьера с таким id нет'\nE         \nE         - Курьера с таким id нет\nE         + Курьера с таким id нет.\nE         ?                       +\n\nAPI-tests/test_scooter_api.py:335: AssertionError"}, "description": "Проверка, что система вернёт ошибку, если указать несуществующий ID курьера", "steps": [{"name": "Отправляем DELETE-запрос на удаление курьера с несуществующим ID", "status": "failed", "statusDetails": {"message": "AssertionError: Сообщение об ошибке не совпадает\nassert 'Курьера с таким id нет.' == 'Курьера с таким id нет'\n  \n  - Курьера с таким id нет\n  + Курьера с таким id нет.\n  ?                       +\n", "trace": "  File \"/Users/vladvihrov/PycharmProjects/YAProjectSprint_7/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/vladvihrov/PycharmProjects/YAProjectSprint_7/API-tests/test_scooter_api.py\", line 335, in test_delete_courier_with_nonexistent_id\n    assert (\n"}, "start": 1725214482487, "stop": 1725214482878}], "attachments": [{"name": "stdout", "source": "19bd1135-d5d9-424f-827c-f101df679f29-attachment.txt", "type": "text/plain"}], "start": 1725214482487, "stop": 1725214482879, "uuid": "4c2de833-cbd5-4aa9-8da0-fa760488efe2", "historyId": "847a08640c0e672cf5ca54165fbc9c6b", "testCaseId": "847a08640c0e672cf5ca54165fbc9c6b", "fullName": "API-tests.test_scooter_api.TestCourierDeletionAPI#test_delete_courier_with_nonexistent_id", "labels": [{"name": "feature", "value": "Удалить курьера"}, {"name": "story", "value": "2. Если отправить запрос с несуществующим id, вернётся ошибка."}, {"name": "parentSuite", "value": "API-tests"}, {"name": "suite", "value": "test_scooter_api"}, {"name": "subSuite", "value": "TestCourierDeletionAPI"}, {"name": "host", "value": "MacBook-Pro-Vlad.local"}, {"name": "thread", "value": "18149-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "API-tests.test_scooter_api"}]}